src/status_im2/contexts/onboarding/common/carousel/animation.cljs

```diff
@@ -33,22 +31,31 @@
      -1)
     paused)))
 
-(defn initialize-animation
-  []
+(defn use-initialize-animation
+  [progress paused animate?]
   (reset! progress (reanimated/use-shared-value 0))
   (reset! paused (reanimated/use-shared-value false))
-  (animate-progress @progress @paused))
+  (rn/use-effect
+   (fn []
+     (animate-progress @progress @paused))
+   [animate?]))
+
+(defn cleanup-animation
+  [progress paused]
+  (fn []
+    (reanimated/cancel-animation @progress)
+    (reanimated/cancel-animation @paused)))
```

Comment:
QA found some performance issues after the Identifiers screens. I figure we want to clear these animations after onboarding because they are not being used. 
Adding this and refactoring the functional components to the new implementation seemed to address the issue so not 100% sure of which is needed. 
Is there any reason we should not cleanup the animations after we are done with them?

Reply:
> Is there any reason we should not cleanup the animations after we are done with them?

If it fixes the issue, there is no harm :)
Just a little curious, because as we have seen in the past once the view gets destroyed animation also stopped.
Example: https://github.com/status-im/status-mobile/issues/15522